charts/surrealdb/Chart.yaml

```
@@ -1,15 +1,15 @@
 apiVersion: v2
 name: surrealdb
 type: application
-version: 0.2.0
+version: 0.3.0
 appVersion: 1.0.0-beta.10
```

now that we are at it...
```suggestion
appVersion: 1.0.0
```